drivers/vulkan/vulkan_context.cpp

```diff
@@ -1332,9 +1332,20 @@ Error VulkanContext::_create_physical_device(VkSurfaceKHR p_surface) {
 	//  features based on this query
 	vkGetPhysicalDeviceFeatures(gpu, &physical_device_features);
 
-	// Check required features
-	ERR_FAIL_COND_V_MSG(!physical_device_features.imageCubeArray, ERR_CANT_CREATE, "Your GPU doesn't support image cube arrays which are required to use the Vulkan-based renderers in Godot.");
-	ERR_FAIL_COND_V_MSG(!physical_device_features.independentBlend, ERR_CANT_CREATE, "Your GPU doesn't support independentBlend which is required to use the Vulkan-based renderers in Godot.");
+	// Check required features and abort if any of them is missing.
+	if (!physical_device_features.imageCubeArray || !physical_device_features.independentBlend) {
+		String error_string = "Your GPU does not support the following features which are required to use Vulkan-based renderers in Godot:\n\n";
+		if (!physical_device_features.imageCubeArray) {
+			error_string += "- No support for image cube arrays.\n";
+		}
+		if (!physical_device_features.independentBlend) {
+			error_string += "- No support for independentBlend.\n";
+		}
+		error_string += "\nThis is usually a hardware limitation, so updating graphics drivers won't help in most cases.";
+
+		OS::get_singleton()->alert(error_string + "\nClick OK to exit.");
+		CRASH_NOW_MSG(error_string);
```

Comment:
Do we really need to crash? Why not return `ERR_CANT_CREATE` and let it fail "properly"?

Reply:
According to https://github.com/godotengine/godot/issues/70013, using `ERR_CANT_CREATE` will end on a black screen that never goes away on mobile platforms.